x-pack/solutions/security/plugins/security_solution/public/common/components/with_data_view/index.tsx

```diff
@@ -30,10 +34,19 @@ export const withDataView = <P extends WithDataViewArg>(
   fallback?: ReactElement
 ) => {
```

Comment:
Nothing wrong with this pattern, but do you think using something like context would be better? Just a quick note, but still working through the code